### PR TITLE
Fixes #6673: No reporting on agents on ncf directives on SLES

### PR DESCRIPTION
**Pull request by amousset**
http://www.rudder-project.org/redmine/issues/6673

### CODE REVIEW

**Comment by amousset** [comment]
Still WIP.
Couldn't we make a compulsory 2.10 step when upgrading from rudder<2.10 to rudder>2.10 and remove old (2.3->2.4 for example) migration steps from new packages?

**Comment by amousset** [comment]
PR updated.

File: rudder-webapp/SOURCES/rudder-upgrade [hunk 1]
```diff
@@ -702,21 +526,277 @@ rudder.server-roles.relay-top=autodetect
...
+################################################################################
+# End
+################################################################################
 
 # For every upgrade, we force the root server to run a new inventory on the next CFEngine run
 touch /opt/rudder/etc/force_inventory
 
 echo ""
-echo "INFO: The migration has completed successfully."
+if $UNCOMPLETE_MIGRATION;
```
Comment (Kegeruneku) [inline rudder-webapp/SOURCES/rudder-upgrade]: The ; is unneeded, and you should use ${UNCOMPLETE_MIGRATION}

**Comment by amousset** [comment]
PR updated.

**Comment by amousset** [comment]
PR updated.

**Comment by Kegeruneku** [comment]
Accepted !

**Pull request merged**